### ISSUE TITLE
tests: script/race fails on aarch64 under heavy load

### ISSUE THREAD

**Issue by mbgg** [karelzak/util-linux#296]
Hi all,

I was able to reliably reproduce a fail of test script "race" on aarch64 under heavy load. Without load the test passes. I wasn't able to reproduce this on my x86_64 installation though.

Anyway I think we should regard reverting:
bd84241 (Revert "tests: start script/race test only when --force specified") from @rudimeier 

Any comments or hints on how to hunt this bug?

**Comment by rudimeier** [comment]
I can confirm problems even for x86_64.

Since a few months I saw some (1-2%) failures again on opensuse build farm (OBS) on any archs. But from the past I have about 9000(!) 100% successful runs logged with util-linux versions between 2.26.git419.7dbcd and v2.27-rc2. [*]

Matthias, could you please try out if you can reproduce the failure with v2.27-rc2 too?

[*] Unfortunately I have no newer log files because I felt guilty to bother OBS farm with 200 automatic UL builds per day ;)

**Comment by mbgg** [comment]
I just tried with v2.27-rc2 and master (7154cc8). Same problem as I described beforehand.
The test fails if the aarch64 system is under heavy load, otherwise works fine.

@rudimeier if you have access to an aarch64 machine in the OBS, then maybe you want to double check. I'm using a local installation in openSUSE leap 42.1. As workload I compile the kernel with the double of threads as the machine has CPUs.
